src/RingGrids/utility_functions.jl

```diff
@@ -1,43 +1,32 @@
-"""
-    true/false = isincreasing(v::Vector)
-
+"""$(TYPEDSIGNATURES)
 Check whether elements of a vector `v` are strictly increasing."""
-function isincreasing(x::Vector)
+function isincreasing(x::AbstractVector)
     is_increasing = true
     for i in 2:length(x)
         is_increasing &= x[i-1] < x[i] ? true : false
```

Comment:
This will not work on GPU 

Could do a 

```Julia
is_increasing = all(diff(x) .> 0) 
``` 
instead

Reply:
Yeah, it's not even used at the moment. Don't remember where it was necessary, but that piece of code is gone.